### PR TITLE
Dtso 2552 dyna

### PR DESCRIPTION
**Pull request by MatthewFox97**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: components/dynatrace_activegate/prod.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
-module "dynatrace_activegate" {
-  source         = "../../modules/dynatrace-activegate"
-  instance_count = 3
-  network_zone   = "azure.cft"
+# module "dynatrace_activegate" {
+#   source         = "../../modules/dynatrace-activegate"
+#   instance_count = 3
+#   network_zone   = "azure.cft"
```
Comment (cakeben) [inline components/dynatrace_activegate/prod.tf:4]: Hi @MatthewFox97 why is the network zone commented out?

Comment (thomast1906) [reply components/dynatrace_activegate/prod.tf:4]: @cakeben - lets catch up tomorrow with this

Comment (thomast1906) [reply components/dynatrace_activegate/prod.tf:4]: @MatthewFox97 ^

**Comment by thomast1906** [comment]
Closing - PR not required now.

**Pull request closed without merging**